pkg/controller/admissionchecks/provisioning/indexer.go

```diff
@@ -71,13 +71,13 @@ func SetupIndexer(ctx context.Context, indexer client.FieldIndexer) error {
 	return nil
 }
 
-func ServerSupportsProvisioningRequest(mgr manager.Manager) bool {
+func ServerSupportsProvisioningRequest(mgr manager.Manager) error {
```

Comment:
Ideally, we want to implement a unit testing since this is exposed to the external package.
But, I think that we can consider it as a follow-ups since we currently do not have any testings against this.

@mimowo What is your thought?

Reply:
+1, but I'm ok with a follow up for this since they didnt exist before